realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -399,26 +398,26 @@ protected final void copy(ColumnInfo rawSrc, ColumnInfo rawDst) {
     }
 
     public static RealmObjectSchema createRealmObjectSchema(RealmSchema realmSchema) {
+        if (realmSchema.contains("AllTypes")) {
```

Comment:
Yes, if the cyclic model classes, and `createRealmObjectSchema()` calls itself. @beeender has a PR which changes that.